parser/jsonquery_visitor_impl.go

```diff
@@ -197,23 +198,23 @@ func (j *JsonQueryVisitorImpl) VisitMulSumExp(ctx *MulSumExpContext) interface{}
 		case ErrEvalOperandMissing:
 			j.setDebugErr(
 				newNestedError(err, "Eval operand missing in input object").Set(ErrVals{
-					"attr_path": ctx.AttrPath().GetText(),
+					"attr_path": "",
```

Comment:
en este error el path creo que sería este ` ctx.ListAttrPaths().GetText(),`

Reply:
you're right.
it has been fixed